addons/l10n_sa_edi/models/account_journal.py

```diff
@@ -4,16 +4,10 @@
 from lxml import etree
 from datetime import datetime
 from base64 import b64encode, b64decode
-from odoo import models, fields, service, _, api
+from odoo import models, fields, _
```

Comment:
As you're changing the import, may I ask you to reorder it properly?
[Guidelines](https://www.odoo.com/documentation/master/contributing/development/coding_guidelines.html#imports:~:text=from%20odoo%20import%20api%2C%20fields%2C%20models%2C%20_)
:nerd_face: